cime_config/namelist_definition_drv.xml

```diff
@@ -2109,43 +2109,47 @@
     <type>char</type>
     <category>aux_hist</category>
     <group>MED_attributes</group>
-    <desc>Auxiliary mediator wav2med file1 output fields (colon delimited)</desc>
+    <desc>Auxiliary mediator wav2med average history output every day.</desc>
     <values>
-      <value>Sw_hs:Sw_wlm:Sw_thm:Sw_thp0:Sw_fp0:Sw_u:Sw_v:Sw_ustokes:Sw_vstokes:Sw_tusx:Sw_tusy</value>
+      <value>Sw_hs:Sw_Tm1:Sw_thm:Sw_thp0:Sw_fp0:Sw_u:Sw_v:Sw_ustokes:Sw_vstokes:Sw_tusx:Sw_tusy</value>
     </values>
+    <desc>Auxiliary mediator wav2med file1 output fields (colon delimited)</desc>
   </entry>
   <entry id="histaux_wav2med_file1_history_option">
     <type>char</type>
     <category>aux_hist</category>
     <group>MED_attributes</group>
-    <desc>Auxiliary mediator wav2med file1 output option</desc>
+    <desc>history option type</desc>
```

Comment:
I think the old description was, well, more descriptive? Why the change?
I would rather either have a better id (e.g., `id="histaux_wav2med_file1_history_freq_option"`) or a better description (or, ideally, both).

Reply:
I have updated the descriptions - but there is now only one auxiliary file (see my comment in the PR description).